pitft_114r_st7789.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
```

Comment:
Now I realize that the naming of these adafruit specific files has not been that consistent. Would you mind renaming this file and the two existing files to have `adafruit_` prefix? (like waveshare_, tontec_ and freeplaytech_ also have). I.e.

````
adafruit_pitft_114r_st7789.h
adafruit_pitft_28r_ili9341.h
adafruit_pitft_35r_hx8357d.h
```